test/snippet/core/type_list/list_traits_drop.cpp

```diff
@@ -0,0 +1,9 @@
+#include <seqan3/core/type_list/traits.hpp>
+
+int main()
+{
+    using list_t = seqan3::type_list<int, float, bool, int>;
+
+    // Drop the first two types in t.
```

Comment:
```suggestion
    // Drop the first two types in list_t.
```
check other places too